draft-lenders-core-coap-dtls-svcb.md

```diff
@@ -60,61 +61,52 @@ normative:
 informative:
   RFC8323: coap-tcp
   I-D.ietf-core-dns-over-coap: doc
+  RFC4944: 6lo
 
 --- abstract
 
-This document specifies the usage of Service Parameters as used in SVCB ("Service Binding") DNS resource
-records for the discovery of transport-layer-secured CoAP services.
+This document specifies an Application-Layer Protocol Negotiation (ALPN) ID for
+transport-layer-secured CoAP services.
 
 --- middle
 
 # Introduction
 
-{{-svcb}} specifies the "SVCB" ("Service Binding") DNS resource records for looking up
-communication endpoints of a service. Service Parameters (SvcParams) are used to
-carry that information.
-This document specifies which information from SvcParams can be used with CoAP services that are
-secured by transport security, namely TLS and DTLS.
-As an example, this information can be obtained as part of the discovery of DNS over CoAP (DoC) servers (see
-{{-doc}}) that deploy TLS or DTLS to secure their messages.
+Application-Layer Protocol Negotiation (ALPN) allows agreeing on an application-layer protocol during a Transport Layer Security (TLS) handshake.
+For that an ALPN ID is used.
+Furthermore, the ALPN ID can be used to discover Service Bindings (SVCB) via DNS, using SVCB resource records with the "alpn" Service Parameter Keys.
```

Comment:
```suggestion
Furthermore, the ALPN ID can discovered as part of Service Bindings (SVCB) via DNS, using SVCB resource records with the "alpn" Service Parameter Keys.
```

AIU they are only a result of the lookup, not part of the lookup. (The service identifier, eg. _dns, is used to discover the binding).